app/views/instruments/schedule.html.haml

```diff
@@ -1,40 +1,84 @@
 = content_for :head_content do
-  = render partial: 'shared/headers/calendar'
-  = javascript_include_tag 'instruments.js'
+  = render "shared/headers/calendar"
+  = javascript_include_tag "instruments.js"
+
 = content_for :h1 do
-  =h current_facility
+  = current_facility
+
 = content_for :sidebar do
-  = render partial: 'admin/shared/sidenav_product', locals: { sidenav_tab: 'instruments' }
+  = render "admin/shared/sidenav_product", sidenav_tab: "instruments"
+
 = content_for :tabnav do
-  = render partial: 'admin/shared/tabnav_product', locals: { secondary_tab: 'reservations'}
+  = render "admin/shared/tabnav_product", secondary_tab: "reservations"
 
 :javascript
   var events_path = "#{facility_instrument_reservations_path(current_facility, @instrument, format: 'js', with_details: true, all: true)}";
-  var withDetails = true
+  var withDetails = true;
 
 %h2= @instrument
 
-%h3 Administrative Reservations
+%h3= t(".administrative_reservations")
 
-%p= link_to 'Add Admin Reservation', new_facility_instrument_reservation_path, class: 'btn-add'
+%p
+  = link_to t(".add_admin_reservation"),
+    new_facility_instrument_reservation_path,
+    class: "btn-add"
+
+  - if current_user.administrator? # TODO: restrict to global admins until the offline feature is complete
+    - if @instrument.online?
+      = link_to t(".mark_instrument_offline"),
+        new_facility_instrument_offline_reservation_path(current_facility, @instrument),
+        class: "btn-add"
```

Comment:
I feel like this button should use a different styling. They kind of blur together. It also might be nice to separate them with some space. Possibly float this one over to the right?